Makefile

```diff
@@ -148,12 +149,15 @@ build: install build-$(CALYPSO_ENV)
 
 build-css: public/style.css public/style-rtl.css public/style-debug.css public/editor.css
 
-build-development: build-server $(CLIENT_CONFIG_FILE) server/devdocs/search-index.js build-css
+build-inline-js:
+	@$(UGLIFYJS) -o public/catch-js-errors.js catch-js-errors.js
```

Comment:
The current build process puts the files into one bundle. This needs to be a separate file for the reasons @blowery mentions here: https://github.com/Automattic/wp-calypso/pull/1868#issuecomment-166304208